nemo/lightning/pytorch/plugins/mixed_precision.py

```diff
@@ -141,4 +199,17 @@ def forward_context(self) -> Generator[None, None, None]:
             pass
 
 
+def patch_dtype_config(dtype_config, config):
```

Comment:
nit: naming doesn't make sense, dtype_config is not the thing being patched

Reply:
Renamed this to `update_config_with_dtype_overrides`. Feel free to let me know if you would like something different.